### DISCUSSION TITLE
HW 2 Part 1 1B

### DISCUSSION THREAD

**Discussion by FishMacaron** [cooperrc/computational-mechanics#35] [Q&A] [answered]
For some reason I keep getting the following error when trying to find the length of my column despite it being the same format as the example from the text? Both of them seem to be series types so I'm confused so to why mine has an error and the example from the text does not...

![image](https://user-images.githubusercontent.com/68212585/172447297-61516065-d5df-413f-ba18-f4013e9f3277.png)
![image](https://user-images.githubusercontent.com/68212585/172447390-b34a6079-de5d-444f-a9ec-90d1fe5edb06.png)
![image](https://user-images.githubusercontent.com/68212585/172447448-c73b9d0d-96bb-43cf-b844-6f4ac5c89aee.png)

**Comment by cooperrc** [comment, marked as answer]
```python

----> len(numot)

TypeError: 'Series' object is not callable
```

This means your function `len` was overwritten by something else (you can check with the command, `whos`). 

Quick fix: restart the kernel and just run the cells you're working on. Look for a line that has `len  = ...`
